src/GovUk.Education.ExploreEducationStatistics.Data.Importer/Utils/DbUtils.cs

```diff
@@ -17,44 +14,5 @@ public static StatisticsDbContext CreateStatisticsDbContext()
 
             return new StatisticsDbContext(optionsBuilder.Options);
         }
-        
-        public static ContentDbContext CreateContentDbContext()
-        {
-            var optionsBuilder = new DbContextOptionsBuilder<ContentDbContext>();
-            optionsBuilder.UseSqlServer(ConnectionUtils.GetAzureSqlConnectionString("ContentDb"),
-                providerOptions => providerOptions.EnableRetryOnFailure());
-
-            return new ContentDbContext(optionsBuilder.Options);
-        }
-
-        /**
-         * Obtains an exclusive lock within a new transaction and stops other transactions needing to acquire the
-         * same lock to run until they can obtain the lock.  The lock is released upon a transaction being committed
-         * or rolled back.
-         */
-        public static Task<TResult> ExecuteWithExclusiveLock<TDbContext, TResult>(
```

Comment:
Why have we gotten rid of all this code? I'm sure @duncan-at-hiveit put this all in to avoid importer threading errors.

Reply:
I think it was only being used when the Theme/Topic/Publication/Release were being created/updated on the stats DB. So because all that is being created before the importer starts, it's no longer used.

Reply:
Ah good point, nice! :100: